src/utils/generator.js

```diff
@@ -176,6 +180,7 @@ type ModifiedPropertyEntryArgs = {|
 class ModifiedPropertyEntry extends GeneratorEntry {
   constructor(args: ModifiedPropertyEntryArgs) {
     super();
+    this.isTerminal = false;
```

Comment:
Why the assignment?

Reply:
Because it's not assigned in the `super` constructor. A bit dirty, but results in the least amount of code...